src/components/ecosystems/Collections/CollectionsContainer.jsx

```diff
@@ -185,6 +186,31 @@ export default compose(
           })
       }
     },
+    duplicateItem: ({ collectionsApiRoute, jekyll, branch, match, activeItem, updateCachedTree, history, collectionsRoute }) => () => {
+      const { collection } = match['params']
+    
+      if (collection && branch) {
+    
+        return jekyll.collection(collection, { ref: branch })
+          .then(collection => {
+            const duplicatedItem = activeItem
```

Comment:
@heislercreative we should use Object.assign() here to ensure we make a copy of the item, otherwise we're editing an existing item directly

Reply:
@5290charlie Thanks, done.